extensions/cornerstone/src/commandsModule.ts

```diff
@@ -72,7 +96,129 @@ const commandsModule = ({ servicesManager }) => {
     return toolGroup;
   }
 
+  function _showViewerContextMenu(viewerElement, options) {
+    let defaultPointsPosition = [];
+    if (options.nearbyToolData) {
+      defaultPointsPosition = commandsManager.runCommand(
+        'getToolDataActiveCanvasPoints',
+        { toolData: options.nearbyToolData }
+      );
+    }
+
+    contextMenuController.showContextMenu(
+      options,
+      viewerElement,
+      defaultPointsPosition
+    );
+  }
+
   const actions = {
+    /** Show the specified context menu */
+    showViewerContextMenu: providedOptions => {
```

Comment:
also I prefer options, and later when you destruct `optionToUse` we use this naming pattern a lot through out the codebase both here, and in cs3d. vtk also uses it which is nice